scripts/HOWTO-distribution.md.bak

```diff
@@ -0,0 +1,777 @@
+# How we build and release JMRI distributions
```

Comment:
Is this file intended to be included in this PR?

Reply:
No.  And I'm surprised it got included, as I (incorrectly) thought it was excluded by .gitignore.  Will fix both parts.